ecl/hqlcpp/hqltcppc.cpp

```diff
@@ -1387,19 +1387,10 @@ void CIfBlockInfo::buildSerialize(HqlCppTranslator & translator, BuildCtx & ctx,
 
 bool CIfBlockInfo::buildReadAhead(HqlCppTranslator & translator, BuildCtx & ctx, ReadAheadState & state)
 {
-    try
-    {
-        OwnedHqlExpr mappedCondition = state.selector->queryRootRow()->bindToRow(condition, queryRootSelf());
-        BuildCtx condctx(ctx);
-        translator.buildFilter(condctx, mappedCondition);
-        return CContainerInfo::buildReadAhead(translator, condctx, state);
-    }
-    catch (IException * e)
```

Comment:
Why is it no longer necessary to catch exceptions from this code block?

Reply:
Previously it was hiding the exceptions - now they will go to the caller and be properly handled (though I think the theory is that there should not be any anyway)